build-parent/pom.xml

```diff
@@ -248,6 +252,42 @@
         <version>${version.org.jboss.shrinkwrap.descriptors}</version>
       </dependency>
 
+      <dependency>
```

Comment:
Why are all these aether dependencies needed here? I didn't see them added anywhere

Reply:
Hmm, I don't recall adding this. I'll check.

Reply:
Got it. Previously, Eclipse Aether was managed in `wildfly-core-parent`, but that's no longer the case. So we have to manage it ourselves.

Reply:
Yes, that is exactly why I added them, sorry, forgot I did it

Reply:
Ok, thanks for clarification